packages/create-react-app/createReactApp.js

```diff
@@ -121,15 +121,28 @@ const program = new commander.Command(packageJson.name)
   })
   .parse(process.argv);
 
+if (program.info) {
+  console.log(chalk.bold('\nEnvironment Info:'));
+  return envinfo
+    .run(
+      {
+        System: ['OS', 'CPU'],
+        Binaries: ['Node', 'npm', 'Yarn'],
+        Browsers: ['Chrome', 'Firefox', 'Safari'],
```

Comment:
Can we add Edge and IE here to support Windows users?

Reply:
I am 100,000% not a windows user - I did my best, and as of envinfo@5.4.0, they should both be there. If someone could test this out `npx envinfo@5.4.0 --browsers`

This shows up for me on a vagrant box.
```
  Browsers:
    Edge: 20.10240.17146.0
    Internet Explorer: 11.0.10240.17443
```
Will update this PR too.

Reply:
It works on Windows 10. And outputs

```
Browsers:
    Edge: 41.16299.371.0
    Internet Explorer: 11.0.16299.371
```

Reply:
awesome thanks for testing - Ill update the PR to the latest version now.